src/features/auth/helpers/contactSupport.ts

```diff
@@ -32,4 +32,4 @@ export const contactSupport: Record<string, ExternalNavigationProps['externalNav
     onSuccess: () => analytics.logMailTo('forPhoneNumberConfirmation'),
     onError: () => eventMonitoring.logError(new ContactSupportError('PhoneNumberConfirmation')),
   },
-}
+} as const satisfies any
```

Comment:
On veut pas justement, enlever le typage ligne 15, et mettre ici : 
```suggestion
} as const satisfies Record<string, ExternalNavigationProps['externalNav']>
```
?

Reply:
ça c'est ton problème ensuite 😂
Le but de cette PR est de faire passer le `satisfies` dans les tests de CI

Cette PR a été faite pour que tu puisse merge ta PR https://github.com/pass-culture/pass-culture-app-native/pull/6231/files sans rentrer dans l'implémentation.

Le but c'est que tu n'ais qu'à résoudre ce conflit.

techniquement ça devrait être remplacé par `as const satisfies ContactSupport` qui est importé d'un autre fichier et là on rentrerait dans l'implémentation

Reply:
Oui oui, perso, c'était pour éviter d'ajouter un any dans le code, c'est tout 🙃